docs/admin/searx.favicons.rst

```diff
@@ -0,0 +1,249 @@
+.. _favicons:
+
+========
+Favicons
+========
+
+.. sidebar:: warning
+
+   Don't activate the favicons before reading the documentation.
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: entry
+
+Activating the favicons in SearXNG is very easy, but this **generates a
+significantly higher load** in the communication between client and server and
+additional resources are used on the server.
+
+To mitigate these disadvantages, various methods have been implemented,
+including a *cache*.  The cache must be parameterized according to your own
+requirements and maintained regularly.
+
+To activate favicons in SearXNG's result list, set a default favicons-resolver
+in the :ref:`search <settings search>` settings:
+
+.. code:: yaml
+
+   search:
+     favicon_resolver: "duckduckgo"
+
+By default and without any extensions, SearXNG serves these resolvers:
+
+- ``duckduckgo``
+- ``allesedv``
+- ``google``
+- ``yandex``
+
+With the above setting favicons are displayed, the user has the option to
+deactivate this feature in his settings.  If the user is to have the option of
+selecting from several *resolvers*, a further setting is required / *but this will
+be discussed later, first we have to setup the favicons cache*.
```

Comment:
```suggestion
With the above setting, favicons are displayed by default. Users have the option to
deactivate this feature in their settings..
```
Forgot to remove? / TODO?

Reply:
Nope .. "but this will be discussed later" means later in this article .. but I [fixed](https://github.com/searxng/searxng/compare/a76b37fe75f50c8281fbec24499e74211ba2b67e..7c7482fc107484e883803df65176647d58aff989) this part to be more clear.